docs/send-data/opentelemetry-collector/remote-management/overview.md

```diff
@@ -0,0 +1,76 @@
+---
+id: overview
+title: OpenTelemetry Remote Management Overview
+sidebar_label: Overview
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Sumo Logic Distribution for OpenTelemetry Collector simplifies remote management of data collection, allowing setup from the Sumo Logic UI and deployment to multiple collectors.
+
+## Key benefits of Remote Management
```

Comment:
I see "Remote Management" and "remote management" used in this article. I think you should change them all to "remote management".